MMDevice/MMDevice.h

```diff
@@ -1423,8 +1423,8 @@ namespace MM {
        */
       virtual int InsertImage(const Device* caller, const unsigned char* buf, unsigned width, unsigned height, unsigned byteDepth, const char* serializedMetadata, const bool doProcess = true) = 0;
 
-      virtual void ClearImageBuffer(const Device* caller) = 0;
-      virtual bool InitializeImageBuffer(unsigned channels, unsigned slices, unsigned int w, unsigned int h, unsigned int pixDepth) = 0;
+      MM_DEPRECATED(virtual void ClearImageBuffer(const Device* caller)) = 0;
+      MM_DEPRECATED(virtual bool InitializeImageBuffer(unsigned channels, unsigned slices, unsigned int w, unsigned int h, unsigned int pixDepth)) = 0;
```

Comment:
`InitializeImageBuffer` serves a purpose that is unrelated to Live mode "continue by overwriting", so should not be deprecated in this PR.

It exists to allow cameras to trigger the memory allocation (which can take time) before opening the shutter, in order to minimize unnecessary exposure. Cameras that don't call it in `StartSequenceAcquisition()` are technically buggy.